packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/android/create-app.md

```diff
@@ -1,3 +1,3 @@
 The [Android quickstart](android link here) teaches you the basics of building an Android application.
```

Comment:
do we have a link for this Android quickstart to build an Android app?

Reply:
We have a sample that is part of the SDK repository https://github.com/okta/okta-oidc-android/tree/master/app

But this sample is more of a driver for testing the functionality of the SDK. It contains advance configurations that might not be suitable for a quick start guide.